addons/website_slides_survey/models/slide_slide.py

```diff
@@ -45,6 +46,29 @@ class Slide(models.Model):
         ('check_certification_preview', "CHECK(slide_type != 'certification' OR is_preview = False)", "A slide of type certification cannot be previewed."),
     ]
 
+    @api.constrains
+    def _check_certification_no_quiz(self):
+        for slide in self:
+            if slide.slide_type == 'certification' and slide.question_ids:
```

Comment:
`if any(slide.slide_type == 'certification' and slide.question_ids for slide in self)`